storage/innobase/buf/buf0flu.cc

```diff
@@ -2630,36 +2624,34 @@ buf_get_total_LRU_list_length(void)
 /*********************************************************************//**
 Adjust the desired page cleaner thread sleep time for LRU flushes.  */
 MY_ATTRIBUTE((nonnull))
-static
-void
+static void
 page_cleaner_adapt_lru_sleep_time(
-/*==============================*/
-	ulint*	lru_sleep_time,	/*!< in/out: desired page cleaner thread sleep
-				time for LRU flushes  */
-	ulint	lru_n_flushed) /*!< in: number of flushed in previous batch */
-
+    ut_monotonic_time
+	* lru_sleep_time, /*!< in/out: desired page cleaner thread
+                                      sleep time for LRU flushes  */
+    ulint lru_n_flushed)  /*!< in: number of flushed in previous batch */
```

Comment:
Should not have been touched

Reply:
Result of clang-format of the function signature due to the changed type of lru_sleep_time